config/locales/subscriber_authentication.yml

```diff
@@ -27,3 +27,9 @@ en:
         <p>Emails sometimes take a few minutes to arrive.</p>
         <p>If you do not receive an email soon, check your spam or junk folder.</p>
         <p>If this does not work, <a class="govuk-link govuk-link--no-visited-state" href="https://www.gov.uk/contact/govuk">contact GOV.UK</a> for help.</p>
+    use_your_govuk_account:
+      heading: "Sign in with your GOV.UK account"
+      description_html: |
+        <p class="govuk-body">You’ve set up a GOV.UK account for <strong>%{address}</strong></p>
```

Comment:
I know we do it everywhere... but I stll feel like we need a better way of doing these paragraph chunks with embedded HTML.

It's particularly the wrapping in repeated `<p>` tags

But with that said I can't think up a better way of doing it 🤔 .
- Using individual locale strings into P tags in the HTML assumes that different languages would need three paragraphs to get across the same point (not impossible we might want to actually use these locales for languages at some point).
- We could make this a list, and have the HTML map over the top and inject it into HTML in the view or a helper. But then we've got nested <strong> and <a> tags here too, so we still end up with HTML in our strings.

I think it's especially the idea of having classes in our locale files that seem like it could be easy to miss if we want changes.

Anyway, just pondering out loud, no changes needed I think this is a fit with what we've done everywhere so far!